docker_registry/images.py

```diff
@@ -60,6 +62,16 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def valid_image_id(f):
```

Comment:
Again a vile hack to hide the misery... let's have this in there until further cleanup.